### DISCUSSION TITLE
How to access root dependencies in packages' package.json?

### DISCUSSION THREAD

**Discussion by amirhhashemi** [yarnpkg/berry#3841] [Q&A] [answered]
I installed `jest` in the root of my monorepo so I can test all of my packages in one go. what if I want to have a script in a package to test that specific package? I tried to put this in the `package.json` of my `api` package:
```json
  "scripts": {
    "test": "jest",
  },
```
And I get `command not found: jest` error. Is there a way to do that?

**Comment by arcanis** [comment, marked as answer]
No, `jest` must be a dependency of the script that uses it (we enforce that dependencies are strictly followed, and don't "leak" from one workspace to another). You have some options though, check this doc entry:

https://yarnpkg.com/getting-started/qa/#how-to-share-scripts-between-workspaces
